Makefile

```diff
@@ -31,3 +31,10 @@ migrations-migrate:
 
 email-preview:
 	@docker-compose exec php php bin/console coopcycle:email:preview > /tmp/coopcycle_email_layout.html && open /tmp/coopcycle_email_layout.html
+
+fresh:
+	-docker ps -a -q | xargs docker stop
+	-docker ps -a -q | xargs docker rm
+	-docker rmi $(docker images -q)
```

Comment:
Wouldn't it be simpler to run `docker-compose down`?

Reply:
It didn't even cross my mind that docker-compose would have such a function. 

[Their docs state that](https://docs.docker.com/compose/reference/down/)nNetworks and volumes defined as external are never removed.

I looked over the project and it seems that no volumes are defined as external. I went ahead and fixed the make to docker-compose down, but it seems pointless now.

Reply:
It's cool. This way the `Makefile` acts as an straightforward interface.